backend/tracim_backend/applications/content_file/controller.py

```diff
@@ -119,7 +124,7 @@ def create_file(self, context, request: TracimRequest, hapic_data=None):
                 ) from exc
         content = api.create(
             filename=_file.filename,
-            content_type_slug=FILE_TYPE,
+            content_type_slug=hapic_data.forms.content_type,
```

Comment:
```suggestion
            content_type_slug=hapic_data.forms.content_type,
```
Will it be possible to have a content-type for office-document with this kind of feature or it doesn't make any sense ? (i was thinking about collabora)

Reply:
It's a first step to un-correlate the storage of a content and its type. It could be used to have content-types for office documents but what would you like to achieve with this?